### PR TITLE
Generic-SQL adding oracle test playbook

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26497

## Description
The generic-sql test playbook is failing sometimes, adding a simpler version of it for Oracle db.
 
## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/GenericSQL/TestPlaybooks/generic-sql-oracle.yml [hunk 1]
```diff
@@ -0,0 +1,362 @@
+id: 1725220c-4672-4994-8489-3c3abd855aa4
+version: 2
```
Comment (bakatzir) [inline Packs/GenericSQL/TestPlaybooks/generic-sql-oracle.yml:2]: id == name
version -1

File: Tests/conf.json [hunk 2]
```diff
@@ -639,7 +639,7 @@
         },
         {
             "integrations": "Generic SQL",
-            "playbookID": "generic-sql",
+            "playbookID": "generic-sql-oracle",
```
Comment (bakatzir) [inline Tests/conf.json:642]: what will happen to `generic-sql` do you plan to stop running it?

Comment (ohaim1008) [reply Tests/conf.json:642]: it will run for mysql, one version of mssql (there is another test playbook that run with the other one) and postgesql.

**Comment by ohaim1008** [comment]
@bakatzir doing it here - https://github.com/demisto/content/pull/7999/

**Comment by ohaim1008** [comment]
@glicht Can I have your opinion?

We were having some issues with the oracle test playbook, from time to time it didn't delete a row in the table so I changed it to a simpler test playbook.
Locally (on self-server) I had no issues with either of the test playbooks but on the circle I got:
Screen Shot 2020-07-15 at 12 58 37
I added max_identifier_length=128 to create_engine (I'll change it so it will be added only in case Oracle db is used)

Currently the tests for 5.0 is failing due to an issue that was solved before (https://github.com/demisto/etc/issues/24218), the 'Test' is failing with the encrypted mssql instance, it works fine in 5.5 and 6.0 so I am wondering what could be the problem.

**Comment by glicht** [comment]
> max_identifier_length

Don't change this setting as it isn't correct for older versions. The right thing to do is to disable warnings. I added a commit to deal with the warnings.

**Comment by glicht** [comment]
> Currently the tests for 5.0 is failing due to an issue that was solved before (demisto/etc#24218), the 'Test' is failing with the encrypted mssql instance, it works fine in 5.5 and 6.0 so I am wondering what could be the problem.

I don't think it is failing because of the issue mentioned as the playbook test passes. It is failing after the instance upgrade. I think it was just a temporary network issue.

File: Packs/GenericSQL/TestPlaybooks/generic-sql-oracle.yml [hunk 3]
```diff
@@ -0,0 +1,362 @@
+id: generic-sql-oracle
+version: -1
+contentitemexportablefields:
```
Comment (glicht) [inline Packs/GenericSQL/TestPlaybooks/generic-sql-oracle.yml:3]: I think running demisto-sdk format will remove this section and other un-needed stuff.

**Comment by ohaim1008** [comment]
Failing in 5.0 - tested on self server.

Config a generic sql instance according to the `Microsoft SQL instance using encrypted connection` in `cof.json`.
Test - succeed.
Running `generic-sql-mssql-encrypted-connection` with the same instance- succeed.
No errors or warning were found.

@glicht - please approve force merge.

File: Packs/GenericSQL/ReleaseNotes/1_0_4.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Generic SQL
+- Internal code improvements.
```
Comment (glicht) [inline Packs/GenericSQL/ReleaseNotes/1_0_4.md:4]:
```suggestion
- Disabled Oracle driver warnings.
```

File: Packs/GenericSQL/ReleaseNotes/1_0_4.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Generic SQL
+- Disabled Oracle driver warnings.
```
Comment (amshamah419) [inline Packs/GenericSQL/ReleaseNotes/1_0_4.md:4]:
```suggestion
- Disabled the warnings from the Oracle driver.
```

**Comment by bakatzir** [comment]
Force merging on the 5.0 failure, let's track master and nightly @ohaim1008

**Pull request merged**